helpers/generate_quick_scan_bin_version_strings.sh

```diff
@@ -38,7 +38,7 @@ for STRING_ENTRY in "${STRING_ENTRY_ARR[@]}"; do
 
   echo "[*] Testing SBOM entry ${COMPONENT//::}"
 
-  if [[ "$(grep -h "cpe.*${COMPONENT//::}:" external/nvd-json-data-feeds/* -r | wc -l 2>/dev/null)" -gt 0 ]]; then
+  if [[ "$(rg -I -N "cpe.*${COMPONENT//::}:" external/nvd-json-data-feeds/* | wc -l 2>/dev/null)" -gt 0 ]]; then
```

Comment:
This is a helper script for the host and I would not recommend using rg in such a script as we need to install it on the host first